<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <layout class="QHBoxLayout" name="main_lay">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="attribute_cmb"/>
   </item>
   <item>
    <widget class="QLineEdit" name="pattern_line">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="colorspace_cmb"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
